src/providers/ldap/sdap_async_sudo.c

```diff
@@ -158,8 +158,8 @@ static char *sdap_sudo_build_host_filter(TALLOC_CTX *mem_ctx,
         goto done;
     }
 
-    /* sudoHost is not specified */
-    filter = talloc_asprintf_append_buffer(filter, "(!(%s=*))",
+    /* sudoHost is not specified an it is a cn=defaults rule */
+    filter = talloc_asprintf_append_buffer(filter, "(&(!(%s=*))(cn=defaults))",
```

Comment:
It is correct to hardcode cn here? We have the `ldap_sudorule_name` option..

On the other hand, I have no idea if the sudo LDAP attribute mappings are useful at all..

Reply:
You are right. I pushed new patch.